### ISSUE TITLE
Improve Accessibility: add title and aria-label into iframe

### ISSUE THREAD

**Issue by dfbsx** [cookpete/react-player#1972]
Hello!

During fixing accessibility issues I've found that title and aria-label is missing in <iframe>, can you update it soon? Thanks! :)
